run_brer/run_config.py

```diff
@@ -277,6 +277,13 @@ def __train(self, **kwargs):
             # In the future runs (convergence, production) we need the ABSOLUTE VALUE
             # of alpha.
             current_alpha = context.potentials[i].alpha
+            if current_alpha == 0:
+                alpha_error = input('Alpha value was constrained to 0.0. Is this warning harmless? Y/N ')
```

Comment:
I appreciate the effort at user-friendliness, but this sort of thing doesn't work well in scripted and multiprocessing environments (and would hang an HPC job for the entire allowed job wall time).

We should throw an exception (either by default, or when told to check parameters... but I support by default).

Possibly, we could provide a way to suppress the exception, such as with an optional key word argument like setting `debug=True` or `check_params=False`.